### ISSUE TITLE
A TFKerasPruningCallback for future versions of Optuna

### ISSUE THREAD

**Issue by robertfmath** [optuna/optuna#4740]
### Motivation

Right now, I'm unable to harness the power of Optuna's pruners to prune unpromising trials of my Keras neural network because there's no Keras callback that sends a given end-of-epoch metric (such as the validation loss) to the Trial instance, and thus Optuna's pruners have no "intermediate values" upon which they can act to do the pruning. It seems that in previous versions of Optuna, there was indeed a Keras pruning callback -- first under the name `KerasPruningCallback` and then under `TFKerasPruningCallback` -- but they've both since been deprecated and there is no analogue to be found in the most recent version of Optuna, v3.2.0.

Are there any plans to bring something like this back? Preliminarily, it looks like I can grab [the code for the former `TFKerasPruningCallback`](https://optuna.readthedocs.io/en/v3.1.0/_modules/optuna/integration/tfkeras.html#TFKerasPruningCallback) and use it with no issue in v3.2.0., but it would be great if something like this could come stock in the future and obviate the need to re-define/re-import this class at the start of each project. I'm not sure if there are already plans to reintroduce this (I couldn't find any discussion of it in recent open or closed issues), but I just thought I'd mention it as something that'd be valuable.

Thank you!

### Description

A potential way to implement this new feature is to create an `optuna.integration.tfkeras` module that defines a `TFKerasPruningCallback` class; the class would presumably be similar, if not identical, to the [`TFKerasPruningCallback` class that was in v.3.1.0](https://optuna.readthedocs.io/en/v3.1.0/_modules/optuna/integration/tfkeras.html#TFKerasPruningCallback).

**Comment by nzw0301** [comment]
Integration modules are separated into other repo https://github.com/optuna/optuna-integration as described in https://github.com/optuna/optuna/issues/4484; you can use the callback by using https://github.com/optuna/optuna-integration. So this repo won't include a Keras callback again. If you have a problem with the callback, could you create an issue on https://github.com/optuna/optuna-integration?

**Comment by robertfmath** [comment]
Hi Kento - thanks for clarifying! I didn't know this migration took place. Happily closing this issue!

(And thanks again for spearheading this absolutely wondrous project. The dashboard in particular is a favorite of mine.)
